presto-main/src/main/java/com/facebook/presto/operator/TableWriterOperator.java

```diff
@@ -242,13 +307,17 @@ public Page getOutput()
 
     @Override
     public void close()
+            throws Exception
     {
+        AutoCloseableCloser closer = AutoCloseableCloser.create();
```

Comment:
wrap in try-with-r

Reply:
It is unnecessary. We don't expect the `register()` methods to fail.

Reply:
yea... it's also customary. just a matter a taste

Reply:
yea... it's also customary. just a matter a taste